src/assets/specs/command-api.yaml

```diff
@@ -91,6 +90,10 @@ components:
                   type: boolean
                 flannel:
                   type: boolean
+            suppressSudo:
+              type: boolean
+            experimentalHostResolver:
```

Comment:
I think we are planning to make this default in this release and perhaps rename the property in the next release. Just mentioning here since not sure how easy it is to rename or does the API needs bumping.